### PR TITLE
Buttons badges

### PR DESCRIPTION
**Pull request by varun-Tad**
Created buttons and badges. Please review and let me know what you think

### CODE REVIEW

**Comment by nikhiljugale007** [comment]
Button: Can add some hover effect on solid buttons and  Buttons with icons other than changing the cursor to the pointer. You can use a more dark color or highlight the border.
Badge: Everything looks good

**Comment by varun-Tad** [comment]
@nikhiljugale007 Okay thank you. will add the hover effect.

File: Components/Badges/badge.css [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    padding-top:3px;
+    padding-right:5px;
+    color:var(--secondary-color);
+
+}
+
+
+.badge-active{
+    background-color:var(--badge-active-color);
+}
```
Comment (Naman-Saxena1) [inline Components/Badges/badge.css:68]: You can add box-shadow: 0 0 0 2px var(--background-color); on .badge-active, .badge-busy to get better effect.

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]
If you are removing border from buttons, then it is better to add custom on click animation to indicate that button is getting clicked. And I suggested 1 more change that you can do to improve badge style. Everything else looks fine. 👍

File: Components/Badges/badge.css (hunk 1 above)
Comment (varun-Tad) [reply Components/Badges/badge.css:68]: Okay, will work on it. Thanks, @Naman-Saxena1

**Pull request merged**